cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,553 @@
+                value:
+                    - camcorders
+                - field: name
+                operator: CONTAINS
+                value: Canon
+            actions:
+                - type: set_value
+                field: camera_brand
+                value: canon_brand
+        camera_copy_name_to_model:
+            priority: 0
+            conditions:
+                - field: family.code
+                operator: IN
+                value:
+                    - camcorders
+            actions:
+                - type: copy_value
+                from_field: name
+                to_field: camera_model_name
+
+Enrichment Rule Structure
+-------------------------
+
+Structure’s elements which define a rule are :
+ - rule code (dynamic)
+ - priority*
+ - conditions
+ - field
+ - locale*
+ - scope*
+ - operator
+ - value
+ - actions
+ - type
+
+An enrichment rule is structured as following :
+
+.. code-block:: yaml
+
+    [free rule code]:
+        priority​*:
+        conditions:
+            - field:
+            locale​*:
+            scope​*:
+            operator:
+            value:
+        actions:
+            - type:
+            [Diverse elements according to the action]
+
+Elements with * are optionals.
+
+**Dashes** - ​before element field and after each element contained in value part are mandatory.
+
+**Colon** : ​mandatory after each structure element.
+
+.. warning::
+
+    Rule’s code is free, but has to respect alphanumeric characters, underscore and dashes only accepted
+    and 255 characters maximum.
+
+A priority can be given to a rule. Priority will be considered for rules execution order. Without any given
+priority, rule has a priority =0. The more a priority is high, the more the rule will be executed in priority.
+Therefore, a rule with priority =90 will be executed before rules with priority =0. If no rule has defined priority,
+they will be executed in a "technical" order. (database reading order)
+
+Action’s conditions can be applied on localizable and scopable values in the PIM. In this case, it has
+to be specified using locale and scope elements.
+
+Enrichment Rule Definition
+--------------------------
+
+Available Actions List
+++++++++++++++++++++++
+
+Copy
+____
+
+This action enable to copy an attribute value towards a value of another attributed.
+
+.. warning::
+
+    ​The source attribute and target attribute have to be the same type of
+    attribute. If source attribute is empty, the value "empty" will also be copied.
+
+Two parameters are required and four other are optionals :
+ - from_field: code of the attribute to be copied
+ - from_locale : code of the value’s locale to be copied (optional).
+ - from_scope : code of the value’s channel to be copied (optional).
+ - to_field: code of the attribute toward copying the value
+ - to_locale: code of the local toward copying the value (optional)
+ - to_scope: code of the channel toward copying the value (optional).
+
```

Comment:
For all three, replace "toward copying": attribute/locale/channel code the value will be copied into